src/openzaak/static/libs/prism/prism.css

```diff
@@ -0,0 +1,142 @@
+/* PrismJS 1.19.0
```

Comment:
Including vendored libraries in version control is not something we do anymore. Please add it via [npm](https://www.npmjs.com/package/prismjs)

Sven or I can help you on how to include it into the frontend toolchain.

Reply:
added `prism` via npm